extensions/amp-next-page/0.1/next-page-service.js

```diff
@@ -180,6 +180,14 @@ export class NextPageService {
       }
     }
 
+    // Drop any amp-analytics tags from the child doc. We want to reuse the
+    // parent config instead.
+    const analytics = doc.querySelectorAll('amp-analytics');
+    for (let i = 0; i < analytics.length; i++) {
+      const item = analytics[i];
+      item.parentNode.removeChild(item);
```

Comment:
use `removeElement` from `dom.js` (using common helper functions would end up reducing the compiled code size)

Reply:
Done